.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+	"name": "Ubuntu",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+		"ghcr.io/devcontainers/features/rust:1": {
+			"version": "latest",
+			"profile": "default",
+			"targets": "aarch64-apple-darwin,aarch64-unknown-linux-musl,x86_64-pc-windows-msvc,x86_64-unknown-linux-gnu,x86_64-unknown-linux-musl,x86_64-unknown-none"
```

Comment:
I guess because this is a dev container, its fine to declare a grab-bag of dependencies up-front, to save folks the hassle of running flowey commands with things like --auto-install.

that said, we should be careful about avoiding similar precedents in other more "productized" contexts

Reply:
agreed, maybe i'll leave a comment about this here to note that this is a devcontainer for _dev_ purposes only. Some projects also specify CI runners via container images, but that's not something we're doing today.